### PR TITLE
App setup

### PR DESCRIPTION
**Pull request by Mylo16**
### In this PR:
- I created the splash screen of my Ruby on Rails Budget-App
- I created the sign up and log in pages for the app
- I created the homepage 
- I created the transactions page
- I implemented the new category page
- I implemented the new transaction page
- I wrote unit and integration tests for the most important components
- Attached below is the description video on the app:
https://www.loom.com/share/d7c926d4742d4b6892a8be3af3fe7ad4

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,204 @@
...
+- **[Dynamic routing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- ## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Coming soon...]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
```
Comment (tugsem) [inline README.md]: - [ ] As per the requirement for this capstone, you need to deploy your app using Render, Netlify or GitHub Pages and put the link here. Please do this.

File: README.md [hunk 2]
```diff
@@ -1 +1,204 @@
...
+
+If you like this project please express your appreciation on LinkedIn.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank all those who helped me throughout this project.
+
```
Comment (tugsem) [inline README.md]: - [ ] As per the requirement for this capstone, please give an appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello @Mylo16  👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️app works properly
✔️app tested thoroughly
✔️design guidelines mostly followed
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Mylo16 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Stiched given design ✔️
- Well Stractured code✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**